apps/browser/qml/pages/SettingsPage.qml

```diff
@@ -152,6 +154,9 @@ Page {
                 description: qsTrId("settings_browser-la-close_all_tabs_description")
                 checked: closeAllTabsConfig.value
                 enabled: AccessPolicy.browserEnabled
+                // Margins adjusted to align with other items on the page
+                leftMargin: Theme.horizontalPageMargin + Theme.paddingLarge + _textSwitchGlassCentre
```

Comment:
sorry, where does this paddingLarge come from?

Reply:
It's from the TextSwitch code. The left margin for the toggle item is set there as:
```
68:             left: parent.left; leftMargin: root.leftMargin - Theme.paddingLarge
```

So we have natural centre of the ComboBox icons (fixed):
`Theme.horizontalPageMargin + (permissionIcon.width) / 2)`

Centre for the TextSwitch Glass items (variable):
`root.leftMargin - Theme.paddingLarge + (Theme.itemSizeExtraSmall / 2)`

Equating and rearranging:
`root.leftMargin = Theme.horizontalPageMargin + Theme.paddingLarge + (permissionIcon.width) / 2) - (Theme.itemSizeExtraSmall / 2)`